packages/gap-web-ui/src/components/question-page/CallServiceMethod.tsx

```diff
@@ -51,6 +52,9 @@ export default async function CallServiceMethod<B extends PageBodyResponse, R>(
       typeof redirectTo === 'string' ? redirectTo : redirectTo(result)
     );
   } catch (err: any) {
+    if (handleRequestError) {
+      return handleRequestError(err);
+    }
```

Comment:
This will indeed work but feels excessive to have a prop perform the logic, unless there's good reason we'd have different behaviour between the applicant & admin frontends

Reply:
Would we also not need to replicate this in the individual apps version of this component?

Reply:
Will action this and the move from gap-web-ui -> admin/utils in a separate ticket